src/shell/services/marketing.ts

```diff
@@ -0,0 +1,53 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+import { Announcement } from "./types";
+
+export const marketingApi = createApi({
+  reducerPath: "marketingApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "https://www.zesty.io/" }),
```

Comment:
This needs to be a configuration value that can be different within different environments

Reply:
@shrunyan are you suggesting that this should be pointing to the dev env of the marketing instance if manager-ui is being ran on the dev env, same for stage?